src/components/hotkeys/hotkeys.service.ts

```diff
@@ -7,6 +7,8 @@ const hotkeys = {};
 const hotkeyChangedSource = new Subject();
 const isMac = window.navigator && window.navigator.platform.indexOf('Mac') !== -1;
 
+const ngZone = new NgZone(false);
```

Comment:
In fact, that line is not needed at all and will not work.  The zone for teh call back is set in two places:

1) For the service it is injected: https://github.com/Hypercubed/ngx-ui/blob/d73c0550f0eab83257b0fde5e50fa9415ffdcd90/src/components/hotkeys/hotkeys.service.ts#L146
2) For the `Hotkey` decorator, we cannot inject, so we create a new zone here: https://github.com/Hypercubed/ngx-ui/blob/d73c0550f0eab83257b0fde5e50fa9415ffdcd90/src/components/hotkeys/hotkeys.service.ts#L131